perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -103,11 +107,11 @@ def _RunPing(sending_vm, receiving_vm, receiving_ip, ip_type):
   Returns:
     A list of samples, with one sample for each metric.
   """
-  if not sending_vm.IsReachable(receiving_vm):
+  if ip_type == vm_util.IpAddressSubset.INTERNAL and not sending_vm.IsReachable(receiving_vm):
```

Comment:
Please update the method documentation.

Reply:
updated the documentation to reflect the enum being used for ip_type